src/Uno.UI.Runtime.Skia.X11/X11XamlRootHost.cs

```diff
@@ -25,18 +25,20 @@ internal partial class X11XamlRootHost : IXamlRootHost
 	private const int DefaultColorDepth = 32;
 	private const int FallbackColorDepth = 24;
 
+	// Note For KeyPress/KeyRelease: subscribing on the top window prevents key inputs from hitting when the pointer
+	// is outside the window. https://github.com/unoplatform/uno/issues/19310
```

Comment:
Can you add a manual test that explicitly requires to move the mouse out of the window? even if it's just a single textbox.